src/relations/database_provides.py

```diff
@@ -180,3 +183,28 @@ def reconcile_users(
             if relation not in requested_users:
                 relation.delete_user(shell=shell)
```

Comment:
it may be a good idea to log out users created and users deleted  to the debug-log